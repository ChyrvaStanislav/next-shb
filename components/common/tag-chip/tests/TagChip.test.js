import { render } from '@testing-library/react';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import TagChip from '../index';

const Component = () => (
  <BrowserRouter>
    <TagChip
      label="Some label"
      link="/blog?category=some-category"
      className="some-class"
    />
  </BrowserRouter>
);

describe('TagChip snapshot', () => {
  test('should TagChip match snapshot', () => {
    const { asFragment } = render(<Component />);

    expect(asFragment()).toMatchSnapshot();
  });
});
