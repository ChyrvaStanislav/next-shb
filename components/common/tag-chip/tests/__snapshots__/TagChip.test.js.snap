// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagChip snapshot should TagChip match snapshot 1`] = `
<DocumentFragment>
  <a
    class="greyChip some-class blogTagChip"
    href="/blog?category=some-category"
    target="_blank"
  >
    Some label
  </a>
</DocumentFragment>
`;
